js_modules/dagit/packages/ui/src/components/Tooltip.tsx

```diff
@@ -28,17 +28,21 @@ const overwriteMerge = (destination: any[], source: any[]) => source;
 interface Props extends Tooltip2Props {
   display?: React.CSSProperties['display'];
   canShow?: boolean;
+  useDisabledButtonTooltipFix?: boolean;
```

Comment:
We should use this anytime we wrap a button that can be disabled with a Tooltip

Reply:
(eventually we can get rid of this prop when we test every callsite with this prop set to `true`)